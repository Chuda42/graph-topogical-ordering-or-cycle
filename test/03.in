crear_personas 3
leer_lista_muertes 1 2 2 3 .
leer_lista_convivencias 1 3 .
existe_cronologia
Fin