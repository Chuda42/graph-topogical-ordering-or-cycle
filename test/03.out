1>2>3>4>No hay una cronologÃ­a para estas restricciones.
5>Fin.
